tests/tests/swfs/avm1/super_visibility/test.toml.disable

```diff
@@ -0,0 +1 @@
+num_frames=1
```

Comment:
`test.toml.disable`? If this is a failing test, this should be `known_failure = true`